src/CppUTestExt/MockNamedValue.cpp

```diff
@@ -115,6 +115,8 @@ void MockNamedValue::setObjectPointer(const SimpleString& type, const void* obje
     {
         comparator_ = defaultRepository_->getComparatorForType(type);
         copier_ = defaultRepository_->getCopierForType(type);
+        if(comparator_ && copier_)
```

Comment:
What is the significance of the copier here? Does it cause the same problem? What if I have a comparator, but no copier? Or a copier, and no comparator?